force-app/main/utilities/schema/classes/SObjectRecordTypes.cls

```diff
@@ -1,20 +1,25 @@
 /**
- * @author Gavin Palmer (gavinhughpalmer@gmail.com)
+ * @author Gavin Palmer (gavin.palmer@cloudshiftgroup.com)
  * @version 1.0
  * @description Utility class to help make interacting with an objects record types slightly easier eg cal call
  * SObjectRecordTypes opportunityRecordTypes = new SObjectRecordTypes('Opportunity');
  * Id defaultOpportunityRecordTypeId = opportunityRecordTypes.getDefaultId();
  *
  * 2019-09-24 : Gavin Palmer - Original
+ * 2020-09-22 : Khin Hou - adding option to get a record type developer name from an Id
  **/
 public inherited sharing class SObjectRecordTypes {
     private final Map<String, Schema.RecordTypeInfo> recordTypesByDeveloperName;
+    private final Map<Id, String> recordTypesById;
```

Comment:
@KhinHou I think it would be better to store this with the RecordTypeInfo as the value, that way we could get more information out based on the id if you have it

Reply:
Done!